packages/shared/src/graphql/fragments.ts

```diff
@@ -24,6 +24,26 @@ export const SOURCE_SHORT_INFO_FRAGMENT = gql`
   }
 `;
 
+export const SOURCE_BASE_FRAGMENT = gql`
+  fragment SourceBaseInfo on Source {
+    id
+    active
+    handle
+    name
+    permalink
+    public
+    type
+    description
+    image
+    membersCount
+    currentMember {
+      role
+      referralToken
+      permissions
+    }
```

Comment:
Just be aware this will need the new fragment

Reply:
Not sure that is already available here. We should be able to update this once we make the Squads v4 PR ready.

Reply:
Have fixed merge conflict. Should be updated now.